src/api/app/policies/project_policy.rb

```diff
@@ -1,17 +1,17 @@
 class ProjectPolicy < ApplicationPolicy
   def initialize(user, record)
-    raise Pundit::NotAuthorizedError, 'record does not exist' unless record
+    require_record(record)
     @user = user
     @record = record
   end
 
   def create?
-    return false unless @user
+    require_user(user)
```

Comment:
there is a difference between return false and raising an exception. This is not create! but create?

Reply:
There is no difference. The gem will raise an exception if it return `false`. So it doesn't matter if you raise it first. Check: https://github.com/varvet/pundit/blob/master/lib/pundit.rb#L70